src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -19,13 +14,12 @@ public class PersonBuilder {
     public static final String DEFAULT_NAME = "Amy Bee";
     public static final String DEFAULT_PHONE = "85355255";
     public static final String DEFAULT_EMAIL = "amy@gmail.com";
-    public static final String DEFAULT_ADDRESS = "123, Jurong West Ave 6, #08-111";
+    public static final String DEFAULT_RELATIONSHIP = "Friend";
```

Comment:
Let's have a discussion with the rest on what should be the default relationship. I think "Friend" is an acceptable value, but perhaps we should tailor it to be more family-specific since the context of most relationships as in the app is bloodline related

Reply:
I don't think "Friend" is acceptable right now since it's meant to be a family tree? What do you guys think about just a very broad "Family" default relationship?